confobo/controllers/voting.py

```diff
@@ -20,4 +24,4 @@ def vote(user: User, choice: int, event: Event) -> bool:
     if LOWEST_VOTE <= choice <= HIGHEST_VOTE:
         return voting.save_vote(user, choice, event)
     else:
-        return False
+        raise BadVoteValueError('You can give from {} to {} stars, not {}'.format(LOWEST_VOTE, HIGHEST_VOTE, choice))
```

Comment:
I had my doubts about that as well. Implemented exceptions in order to catch them and return to the user as messages.
Yep, discussion is needed.